.github/workflows/docs-generator-trigger.yml

```
@@ -0,0 +1,30 @@
+name: Docs Generator trigger
+
+on:
+  workflow_run:
+    workflows: [Release]
+    types: [completed]
+    branches: [master]
+
+concurrency:
+  group: "docs"
+  cancel-in-progress: true
+
+jobs:
+  trigger-generator:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

```suggestion
    if: ${{ github.event.workflow_run.conclusion == 'success' }}
    environment: DOCS
```
Can we please add an [environment](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenvironment) here. This will allow us to add secret protection so the secret is only accessible under certain conditions, i.e. when run via `master` and not from Pull Requests.